rush.json

```diff
@@ -19,6 +19,11 @@
       "packageName": "@bentley/imodel-select-react",
       "projectFolder": "packages/imodel-select",
       "versionPolicyName": "imodel-select"
+    },
+    {
+      "packageName": "@bentley/tree-widget-react",
+      "projectFolder": "packages/tree-widget",
+      "versionPolicyName": "tree-widget"

```

Comment:
The `versionPolicyName` here corresponds to the file in `common/config/rush/version-policies.json`. If the model-select-react is within the same major version, you can probably just reuse the one that's already there.

Reply:
Also, don't worry about that pipeline failure. I'll push a fix for that in one second.

Reply:
Does the fact that the version policy for the imodel-select-react package has a locked major version of 2 prevent me from sharing it with the tree-widget react package, which has a major version of 1 currently?

Reply:
Yes, that would prevent it. You'll need to create a new one then.

Reply:
done